ixwebsocket/IXWebSocketPoll.cpp

```diff
@@ -0,0 +1,109 @@
+/*
+ *  IXWebSocketPoll.cpp
+ *  Author: Korchynskyi Dmytro
+ *  Copyright (c) 2017-2018 Machine Zone, Inc. All rights reserved.
+ */
+
+#include "IXWebSocketPoll.h"
+
+namespace ix
+{
+
+    WebSocketPoll::WebSocketPoll(WebSocket* websocket)
+    {
+        bindWebsocket(websocket);
+    }
+
+    WebSocketPoll::~WebSocketPoll()
+    {
+        if (!_messages.empty())
+        {
+            // not handled all messages
+        }
+
+        bindWebsocket(nullptr);
+    }
+
+    void WebSocketPoll::bindWebsocket(WebSocket * websocket)
+    {
```

Comment:
I don't like code that is too indented if I can avoid it. Here I would prefer the early out style:

```
if (_websocket == websocket) return;
```

Reply:
Ok